content/en/docs/languages/python/propagation.md

```diff
@@ -9,12 +9,30 @@ Although not limited to tracing, it is what allows traces to build causal
 information about a system across services that are arbitrarily distributed
 across process and network boundaries.
 
-## Manual W3C Trace Context Propagation
+OpenTelemetry provides a text-based approach to propagate context to remote
+services using the [W3C Trace Context](https://www.w3.org/TR/trace-context/)
+HTTP headers.
 
-The following generic example demonstrates how you can propagate trace context
+## Automatic context propagation
+
+Auto-instrumentation exists for some popular Python frameworks and libraries,
```

Comment:
Since the term "auto-instrumentation" is (apparently) being newly introduced through this PR, should we consider using "zero-code" instead throughout? /cc @svrnm

Reply:
I'm a little bit confused here. Should we use Zero-code when referring to Auto-instrumentation in docs?

Reply:
Especially in this place it's not the right wording, as it is about automatic context propagation. I changed the wording for JS: https://opentelemetry.io/docs/languages/js/propagation/